src/main/java/bio/terra/workspace/db/exception/InvalidWorkspaceStageException.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.workspace.db.exception;
+
+import bio.terra.workspace.common.exception.BadRequestException;
+import java.util.List;
+
+public class InvalidWorkspaceStageException extends BadRequestException {
```

Comment:
do we need all the constructors, or even a separate exception class? I think the message would be enough to figure out what went wrong?

Reply:
Where possible I prefer using more specific exception classes, I think it generally makes errors easier to trace and debug.